src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -76,6 +76,7 @@ abstract class AbstractCloner implements ClonerInterface
         'Exception' => ['Symfony\Component\VarDumper\Caster\ExceptionCaster', 'castException'],
         'Error' => ['Symfony\Component\VarDumper\Caster\ExceptionCaster', 'castError'],
         'Symfony\Component\DependencyInjection\ContainerInterface' => ['Symfony\Component\VarDumper\Caster\StubCaster', 'cutInternals'],
+        'Symfony\Component\EventDispatcher\EventDispatcherInterface' => ['Symfony\Component\VarDumper\Caster\StubCaster', 'cutInternals'],
```

Comment:
Should I update this one and the Container one to use the Contacts namespace ?

Reply:
We know only about implementations of this interface, so let's keep as is.